js/localization/messages/sl.json

```diff
@@ -0,0 +1,295 @@
+
+        "dxScheduler-noTimezoneTitle": "Brez časovnega pasa",
+        "dxScheduler-moreAppointments": "še {0}",
+
+        "dxCalendar-todayButtonText": "Danes",
+        "dxCalendar-ariaWidgetName": "Koledar",
+
+        "dxColorView-ariaRed": "Rdeča",
+        "dxColorView-ariaGreen": "Zelena",
+        "dxColorView-ariaBlue": "Modra",
+        "dxColorView-ariaAlpha": "Prosojno",
+        "dxColorView-ariaHex": "Koda barve (hex)",
+
+        "dxTagBox-selected": "{0} izbranih",
+        "dxTagBox-allSelected": "Vsi izbrani ({0})",
+        "dxTagBox-moreSelected": "še {0}",
+
+        "vizExport-printingButtonText": "Natisni",
+        "vizExport-titleMenuText": "Izvozi/Natisni",
+        "vizExport-exportButtonText": "{0} datoteka",
+
+        "dxFilterBuilder-and": "In",
+        "dxFilterBuilder-or": "Ali",
+        "dxFilterBuilder-notAnd": "In ne",
+        "dxFilterBuilder-notOr": "Ali ne",
+        "dxFilterBuilder-addCondition": "Dodaj pogoj",
+        "dxFilterBuilder-addGroup": "Dodaj skupino",
+        "dxFilterBuilder-enterValueText": "<vnesi vrednost>",
+        "dxFilterBuilder-filterOperationEquals": "Enako",
+        "dxFilterBuilder-filterOperationNotEquals": "Ni enako",
+        "dxFilterBuilder-filterOperationLess": "Je manj kot",
+        "dxFilterBuilder-filterOperationLessOrEquals": "Je manj ali enako kot",
+        "dxFilterBuilder-filterOperationGreater": "Je večje kot",
+        "dxFilterBuilder-filterOperationGreaterOrEquals": "Je večje ali enako kot",
+        "dxFilterBuilder-filterOperationStartsWith": "Se začne z",
+        "dxFilterBuilder-filterOperationContains": "Vsebuje",
+        "dxFilterBuilder-filterOperationNotContains": "Ne vsebuje",
+        "dxFilterBuilder-filterOperationEndsWith": "Se konča z",
+        "dxFilterBuilder-filterOperationIsBlank": "Je prazno",
+        "dxFilterBuilder-filterOperationIsNotBlank": "Ni prazno",
+        "dxFilterBuilder-filterOperationBetween": "Je med",
+        "dxFilterBuilder-filterOperationAnyOf": "Je karkoli izmed",
+        "dxFilterBuilder-filterOperationNoneOf": "Ni nič izmed",
+
+        "dxHtmlEditor-dialogColorCaption": "Zamenjaj barvo pisave",
+        "dxHtmlEditor-dialogBackgroundCaption": "Zamenjaj barvo ozadja",
+        "dxHtmlEditor-dialogLinkCaption": "Dodaj povezavo",
+        "dxHtmlEditor-dialogLinkUrlField": "URL",
+        "dxHtmlEditor-dialogLinkTextField": "Tekst",
+        "dxHtmlEditor-dialogLinkTargetField": "Odpri povezvo v novem oknu",
+        "dxHtmlEditor-dialogImageCaption": "Dodaj sliko",
+        "dxHtmlEditor-dialogImageUrlField": "URL",
+        "dxHtmlEditor-dialogImageAltField": "Dodatno besedilo",
+        "dxHtmlEditor-dialogImageWidthField": "Širina (px)",
+        "dxHtmlEditor-dialogImageHeightField": "Višina (px)",
+        "dxHtmlEditor-heading": "Naslov",
+        "dxHtmlEditor-normalText": "Običajno besedilo",
+
+        "dxFileManager-newFolderName": "",
```

Comment:
@leonkosak We added one more localization message recently. It's related to our File Manager component. Could you please add a translation for `Untitled folder` string here?